Sketchbook/CRSF_Joystick/CRSF_Joystick.ino

```diff
@@ -30,7 +31,7 @@ void packetChannels()
 {
     // Manually expanding instead of looping so I can change params as needed
     
-    // X - Channel 1 - A
+    // A - Channel 1 - X

```

Comment:
Do you have a link to the EdgeTX default mapping?

Reply:
https://github.com/EdgeTX/edgetx/blob/a04a56c0e7a657925b42e8f3e36035a357e5ed5e/radio/src/targets/common/arm/stm32/usbd_hid_joystick.c#L102

https://github.com/EdgeTX/edgetx/blob/a04a56c0e7a657925b42e8f3e36035a357e5ed5e/radio/src/targets/common/arm/stm32/usbd_hid_joystick.c#L127